### ISSUE TITLE
Tmux pane disappears when opening nvim in its neighboring pane 

### ISSUE THREAD

**Issue by agronskiy** [asciinema/asciinema-player#221]
Hello! First, thanks for a wonderful project! So, the issue is as follows:

- the following asciicast [Tmux pane disappears when opening nvim in its neighboring pane - asciinema](https://asciinema.org/a/V4JMoqkoMNotx983nmt63EzLG) consists in opening tmux with two panes, and firing neovim in one of them. Then repeating in another pane.
- **reproducing issue part** when replaying the cast in the web player, the idle tmux pane (i.e. not the one where neovim was fired) disappears, also does a part of the pane separator. The pane reappears when switching to it.
- **expected flow** when replaying in terminal with asciinema play, all works fine -- the panes do not disappear.

Additional info:
- macos 11.5.2 (Big Sur)
- iTerm2 for recording and local replay
- tmux version `3.2a`

**Comment by pocek** [comment]
Likely related to DECSLRM which asciinema-player doesn't support. xterm produces it unless configured as vt220:
```
*.vt100.decTerminalID: vt220
```

**Comment by ku1ik** [comment]
Aha! That brings us closer to having it fixed. Thanks!

**Comment by ku1ik** [comment]
Yeah, pretty sure it's DECSLRM missing in [avt](github.com/asciinema/avt).

Found related issues for other terminal emulators:

- Alacritty - https://github.com/alacritty/alacritty/issues/160
- xterm.js - https://github.com/xtermjs/xterm.js/issues/4285
- Gnome Terminal / Console - https://gitlab.gnome.org/GNOME/vte/-/issues/2526

It seems currently it's supported in xterm (not xterm.js) and iTerm2.
